.github/workflows/regular-release.yaml

```diff
@@ -35,8 +35,8 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.DB_CI_CPROVER_ACCESS_TOKEN }}
           CBMC_TAG: ${{ needs.get-version-information.outputs.version }}
         with:
-          tag_name: ${{ env.CBMC_TAG }}
-          release_name: ${{ env.CBMC_TAG }}
+          tag_name: cbmc-${{ env.CBMC_TAG }}
```

Comment:
why not just have two outputs, one with the version number and one with the full tag name?